images/views.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.generics import (
+    RetrieveUpdateAPIView,
+)
+from rest_framework.parsers import MultiPartParser, FormParser
+
+from profiles.permissions import UserIsProfileOwnerOrReadOnly
+from profiles.models import Profile
+from .serializers import BannerSerializer
+
+
+class BannerChangeAPIView(RetrieveUpdateAPIView):
```

Comment:
I think it's better to rename the view. Something like `BannerRetrieveUpdate` sounds good for me.

Reply:
Thank you for feedback. Done.